client/webserver/site/src/js/markets.js

```diff
@@ -9,6 +9,7 @@ import ws from './ws'
 
 var app
 const bind = Doc.bind
+const disableMouseWheel = Doc.disableMouseWheel
```

Comment:
Since you're only using this function once, I wouldn't personally assign like this, but I don't really have a strong opinion here. Fine as is, if you want.

Reply:
Done.